tasks/src/main/java/org/example/georgeAndAccommodation/GeorgeAndAccommodation.java

```diff
@@ -0,0 +1,29 @@
+package org.example.georgeAndAccommodation;
+
+import java.util.Scanner;
+
+/**
+ * A. George and Accommodation
+ * https://codeforces.com/problemset/problem/467/A
+ */
+public class GeorgeAndAccommodation {
+    public int georgeAndAccommodation() {
+        Scanner sc = new Scanner(System.in);
+
+        int numberOfRooms = sc.nextInt();
+        int numberOfPeople;
+        int capacity;
+        int counter = 0;
+
+        while (numberOfRooms-- >= 1) {
+            numberOfPeople = sc.nextInt();
+            capacity = sc.nextInt();
+            if (capacity - numberOfPeople >= 2) {
+                counter++;
+            }
+        }
+        // Do we always have to close scanner??
```

Comment:
nie musimy zamykac skannera zawsze, jest to dobra praktyka bo oszczedzamy pamiec ale wiele ponad to nie wnosi